solr/conf/solrconfig.xml

```diff
@@ -165,7 +165,8 @@
         inheritable_discover_access_group_ssim,inheritable_discover_access_person_ssim,
         inheritable_read_access_group_ssim,inheritable_read_access_person_ssim,
         inheritable_edit_access_group_ssim,inheritable_edit_access_person_ssim,
-        inheritable_embargo_release_date_dtsi
+        inheritable_embargo_release_date_dtsi,
+        has_model_ssim
```

Comment:
@orangewolf will this change require some kind of migration for existing apps?

Reply:
Summary from Rob (discussed in standup) -- 

Update solr config by sending it to Solr cloud, or editing the file on the server. Not super easy, but we know how to do it. We will need to include instructions for this in the release notes.

Reply:
Tracking on this ticket: https://github.com/scientist-softserv/palni-palci/issues/398